mysql/datadog_checks/mysql/queries.py

```diff
@@ -12,6 +12,10 @@
 ORDER BY `percentile` ASC
 LIMIT 1"""
 
+SQL_QUERY_TABLE_ROWS_STATS = """\
+SELECT   table_schema, table_name, rows_read, rows_changed
+FROM     information_schema.table_statistics"""
```

Comment:
is this the right name for this table? I can only find `information_schema.statistics` in the basic mysql documentation https://dev.mysql.com/doc/refman/8.0/en/information-schema-statistics-table.html

I do see it as a table listed in mariadb docs https://mariadb.com/kb/en/information-schema-table_statistics-table/. Can we just document that this is only available for these types of databases?

Reply:
userstat is a Percona feature both for mysql and mariadb https://www.percona.com/doc/percona-server/8.0/diagnostics/user_stats.html
I change the metric comment to specify the need of a Percona instance

Reply:
I added a small comment please feel free to change if needed